### ISSUE TITLE
Animation is pausing when I am issuing a "sleep" call

### ISSUE THREAD

**Issue by kbessemer** [xvrh/lottie-flutter#269]
The animation is pausing everytime I call sleep, which is necessary for my app at times. Is there a way to have the animation play anyways? It seems to pause when sleep starts, and resume when it stops.

Lottie.asset('assets/loading.json');

sleep(Duration(milliseconds: sleepAmount));

**Comment by xvrh** [comment]
I don't think it is an issue with this library. I guess all animation are paused when you use the [`sleep` function](https://api.dart.dev/stable/2.19.4/dart-io/sleep.html).
I recommend re-working your code and using something like `await Future.delayed(Duration(milliseconds: sleepAmount))`.
